packages/plans-ui/src/plans-grid/index.tsx

```diff
@@ -12,16 +12,15 @@ import { Plans } from '@automattic/data-stores';
 /**
  * Internal dependencies
  */
-import { Title, SubTitle } from '../../titles';
-import ActionButtons from '../../action-buttons';
+import { Title, SubTitle } from '../titles/';
+import ActionButtons from '../action-buttons';
```

Comment:
We can have these in a separate package (we discussed about the concept of a canvas) or at least some common SCSS mixins to reuse between PlansGrid and DomainPicker.
For now in order to reduce duplication I'd say we can just pass the header entirely as a prop since we want to have the content customisable anyway.